app/client/components/pages/businessVerificationPage.jsx

```diff
@@ -44,14 +44,26 @@ class BusinessVerificationPage extends React.Component {
   }
 
   componentDidMount() {
-    this.props.loadBusinessProfile()
+    const { stepStatus, businessProfile } = this.props;
+    let pathname_array = location.pathname.split('/').slice(1);
+    let pathUserId = parseInt(pathname_array[1]);
+    let query;
+    if (!Object.values(businessProfile).length > 0 || pathUserId) {
```

Comment:
why not just `Object.values(businessProfile).length === 0` rather than using a `not` operator?

Reply:
fixed